templates/zerver/help/glossary.md

```diff
@@ -0,0 +1,55 @@
+### customization
+The process of modifying Zulip's settings to fit a user's set of personal preferences. Users can customize their Zulip settings by clicking the cog (![cog](/static/images/help/cog.png)) in the upper right corner of the right sidebar and choose **Settings** from the dropdown menu.
+### emoji
+A small image used to express an idea or emotion in a conversation. Emojis can be included in Zulip messages by  enclosing an emoji phrase (a complete list of emojis can be found [here](http://www.webpagefx.com/tools/emoji-cheat-sheet/)) in colons or clicking the **smiley face** (![smiley face](/static/images/help/smiley-button.png)) icon below the messaging box.
+### home
+A view (or filter) that shows all messages to the user's subscribed streams in chronological order. This view is accessible by clicking the **Home** link in the upper left sidebar.
+### integration
+A tool that combines Zulip with another service or application. Zulip offers a variety of integrations for version control, issue tracker, CI system, monitoring tools, and other tools used for development. The three types of integrations that Zulip supports are webhook integrations, Python script integrations, and plugin integrations.
+### mention
+When you mention users in Zulip, you can type `@` and their email addresses or usernames for auto-completions in your messaging box. The users will be mentioned in your message as `@**username**`, and they will be notified of your message. Users can see all messages they've been messaged in through the **@-mentions** link in the left sidebar or use **is:mentioned** as a narrow or a search constraint.
+### message editing
+If enabled by an administrator, the user has a few minutes to edit the content of their recently-posted message by hovering to the right of their message's timestamp and clicking the pencil (![pencil](/static/images/help/pencil.png)) icon that appears. Zulip labels edited messages with **(EDITED)** next to the username of the message writer.
+### messaging box
+The area where you input the content of your message when you send stream or private messages. The messaging box features several features that can help you compose your message, such as the **Emojis**, **Attach files**, **Formatting**, and **Preview** features.
+### muting
+Muting a topic or stream allows a user to stop being notified of messages sent to the specified topic or stream. This can be activated by clicking the corresponding chevron of the stream or topic and selecting **Mute the [stream/topic] [stream/topic name]** from the dropdown that appears.
+### narrowing
+Narrowing a topic or stream shows only the messages of a particular stream or topic. This can be activated by clicking the corresponding chevron of the stream or topic and selecting **Narrow to [stream/topic] [stream/topic name]** from the dropdown that appears or clicking the conversation topic of a set of messages.
+### notification
```

Comment:
A notification can be:
- Email notification
- Browser notification. It appears for a short time with an excerpt of the message. You also get to hear an alert sound.